src/platform-implementation-js/dom-driver/gmail/views/gmail-compose-view/add-button.ts

```diff
@@ -34,6 +34,14 @@ export default function addButton(
           gmailComposeView.getGmailDriver(),
         );
 
+        if (
+          buttonViewController instanceof DropdownButtonViewController &&
+          buttonOptions?.hasDropdown !== true
+        ) {
+          buttonViewController.destroy();
+          buttonViewController = undefined;
```

Comment:
Because we call `resolve` below this on [line 52](https://github.com/InboxSDK/InboxSDK/pull/1124/files#diff-0cc9418b0cfda04a71c671ef1a2be0fc8bcdaa878ff68eeae10e7820c97b5089L44), this approach might not be workable.

`ComposeButtonView` a few frames higher is only using the initial `Promise.resolve`. Any subsequent `resolve` call after the first is a no-op.